### PR TITLE
add documentation on deprecated easyconfigs/toolchains

### PR DESCRIPTION
**Pull request by boegel**
see also https://github.com/easybuilders/easybuild-framework/pull/2656

### CODE REVIEW

**Comment by boegel** [comment]
@ocaisa Wasn't there yet because there was an issue with readthedocs, but it's there now, see https://boegel-eb.readthedocs.io/en/deprecated_ecs_tcs/Deprecated-easyconfigs.html

File: docs/Deprecated-easyconfigs.rst [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+
+.. _deprecated_easyconfigs_symptoms:
+
+Symptoms
+--------
+
+Using an easyconfig file or toolchain that was marked as deprecated results in a warning message like::
+
+  WARNING: Deprecated functionality, will no longer work in v4.0: easyconfig file '/home/example/test.eb' is marked as deprecated:
```
Comment (ocaisa) [inline docs/Deprecated-easyconfigs.rst]: These blocks are being syntax highlighted (for Python I think). Not a sphinx expert but should be easy to switch off.

Comment (boegel) [reply docs/Deprecated-easyconfigs.rst]: You would expect this to be easy to fix, turns out it's not...

With `.. code:: none`, the block is not displayed at all, with `.. code:: text` I'm still getting highlighting.

I'll keep poking at it, thanks for pointing this out, it indeed looks a bit silly this way...

**Comment by ocaisa** [comment]
Seems to be ok now, if you're ok with it I can merge

**Comment by ocaisa** [comment]
According to https://stackoverflow.com/questions/44376893/selectively-disable-readthedocs-syntax-highlighting you also could have used
``
.. highlight:: none
``

**Comment by boegel** [comment]
@ocaisa Tried `.. highlight:: none` via http://rst.ninjs.org/, and it barfed on it...

Anyway, problem fixed indeed, good to go imho!

**Comment by ocaisa** [comment]
Yeah, I think it's a sphinx option as opposed to an rst option

**Pull request merged**